### PR TITLE
Refactor/2318/migrate unfocus button

### PR DESCRIPTION
**Pull request by shaman-apprentice**
ready for review

### CODE REVIEW

File: visualization/app/codeCharta/state/store.service.ts [hunk 1]
```diff
@@ -65,6 +66,10 @@ export class StoreService {
 			this.dispatch(setIsLoadingMap(true))
 		}
 
+		if (isActionOfType(action.type, IsLoadingFileActions)) {
```
Comment (shaman-apprentice) [inline visualization/app/codeCharta/state/store.service.ts:69]: After migration of nodeContextMenu I will introduce [effects](https://ngrx.io/guide/effects) to get rid of this custom dispatch logic (which should have been a middleware in redux world and will become an effect in ngrx world)

File: visualization/app/codeCharta/state/store/dynamicSettings/dynamicSettings.splitter.ts [hunk 2]
```diff
@@ -44,10 +43,10 @@ export function splitDynamicSettingsActions(payload: RecursivePartial<DynamicSet
 		actions.push(splitSearchPatternAction(payload.searchPattern))
 	}
 
-	if (payload.focusedNodePath !== undefined) {
-		actions.push(splitFocusedNodePathAction(payload.focusedNodePath))
-	}
-
+	// is this used?
```
Comment (shaman-apprentice) [inline visualization/app/codeCharta/state/store/dynamicSettings/dynamicSettings.splitter.ts:46]: I didn't check carefully yet, if this was used. Maybe also a second look? @ce-bo if you have time? :)

Comment (shaman-apprentice) [reply visualization/app/codeCharta/state/store/dynamicSettings/dynamicSettings.splitter.ts:46]: (note at myself: if not used also delete splitFocusedNodePathAction)

Comment (shaman-apprentice) [reply visualization/app/codeCharta/state/store/dynamicSettings/dynamicSettings.splitter.ts]: it was used and is re-added again

**Review by Hall-Ma — APPROVED** [review]
LGTM! 🚀:dizzy:

**Pull request merged**